src/Entity/StudyLevel.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\StudyLevelRepository;
+use Symfony\Component\Validator\Constraints as Assert;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=StudyLevelRepository::class)
+ */
+class StudyLevel
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private int $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $name;
```

Comment:
You probaby should have validation constraints here, because you have a form associated that don't have validations either